pkg/reconciler/serverlessservice/serverlessservice.go

```diff
@@ -215,6 +215,7 @@ func (r *reconciler) reconcilePublicEndpoints(ctx context.Context, sks *netv1alp
 	// The reason for this is, we don't want to leave the public service endpoints empty,
 	// since those endpoints are the ones programmed into the VirtualService.
 	//
+	logger.Debugf(psn+" is in mode %v", sks.Spec.Mode)
```

Comment:
```suggestion
	logger.Debugf("%s is in mode %v", psn, sks.Spec.Mode)
```
Since we're already doing fmt.